site/content/docs/contributing/getting-started.md

```diff
@@ -85,22 +91,27 @@ if there is any existing discussion or work related to your interests.
 
 If you do not see anything, please [file a new issue][file an issue].
 
+> **NOTE**: _Please_ file an enhancement / [feature request issue][fr-issue] to discuss features before filing a PR (ideally even before writing any code), we have a lot to consider with respect to our
+> existing users and future support when accepting any new feature.
+> To streamline the process, please reach out and discuss the concept and design
+> / approach ASAP so the maintainers and community can get involved early.
+
 Please reach out for bugs, feature requests, and other issues!  
 The maintainers of this project are reachable via:
 
-- [Kubernetes Slack] in the [#kind] channel
-- [filing an issue][file an issue]
-- The Kubernetes [SIG-Testing Mailing List]
+- [Kubernetes Slack] in the [#kind] channel (most active, along with the community)
+- The issue tracker by [filing an issue][file an issue]
+- The Kubernetes [SIG-Testing][SIG-Testing] [Mailing List][SIG-Testing Mailing List]
 
 Current maintainers are [@BenTheElder] and [@munnerz] - feel free to
```

Comment:
should I? :smile:

Reply:
you can ping us :-)
or you want to be on the list? that's coming, we'll talk soon :-)

Reply:
as promised https://github.com/kubernetes-sigs/kind/pull/2025